waggle-dance-core/pom.xml

```diff
@@ -11,8 +11,8 @@
 
   <properties>
     <spring-boot-maven-plugin.version>1.5.7.RELEASE</spring-boot-maven-plugin.version>
-    <mockito.version>1.10.19</mockito.version>
-    <powermock.version>1.6.4</powermock.version>
+    <mockito.version>2.8.47</mockito.version>
```

Comment:
Hopefully we'll get @andreeapad's PR with the version changes merged into master soon and we can then merge that in here and get all the version numbers consistent. Until then this is fine.